src/Catalog/Helpers/Utils.cs

```diff
@@ -244,12 +245,22 @@ public static IGraph CreateGraph(string json)
 
         public static IGraph CreateGraph(JToken compacted)
         {
+            return CreateGraph(compacted, readOnly: false);
+        }
+
+        public static IGraph CreateGraph(JToken compacted, bool readOnly)
+        {
             JToken flattened = JsonLdProcessor.Flatten(compacted, new JsonLdOptions());
 
             IRdfReader rdfReader = new JsonLdReader();
-            IGraph graph = new Graph();
+            Graph graph = new Graph();
```

Comment:
nitpick: I think this should stay `IGraph`, as it's the interface rather than the implementation, and this method returns an `IGraph`